### DISCUSSION TITLE
Device support for Simplified Chinese font display

### DISCUSSION THREAD

**Discussion by xiasi0** [HASwitchPlate/openHASP#658] [Q&A] [answered]
have successfully burned the English version of the firmware and it is working properly. I hope it can display Simplified Chinese font

**Comment by fvanroie** [comment, marked as answer]
You can if your device hardware supports PSram and  has a large 16Mb flash to hold the font file.
Upload a Truetype font, for example `chinese.ttf` and use the `text_font` parameter to select it:

![screenshot](https://github.com/HASwitchPlate/openHASP/assets/15969459/96b2419b-dee2-44be-9925-40b3f5b3fc33)

```json
{"page":4,"id":1,"obj":"btn","x":5,"y":35,"w":90,"h":90,"radius":0,"value_str":"\uE769","text":"\n欢迎","toggle":true,"value_ofs_y":-24, "text_font": "chinese36"}
{"page":4,"id":250,"obj":"label","x":240,"y":150,"w":240,"align":"center","template":"祝您有个美好的一天！", "text_font": "chinese24"}
```

**Reply by xiasi0** [reply]
Thank you, fvanroie! According to your method, it has been working properly with Chinese fonts. We look forward to adding Chinese pre compiled firmware to your project!
